### PR TITLE
Palindrome checker updated

### PR DESCRIPTION
**Pull request by Aarti002**
@991rajat can u please review this PR , there were some issue in previous one. And let me know if its not working fine!
Thank you!

### CODE REVIEW

**Comment by 991rajat** [comment]
Is it looking right on your side. Here It's not.

File: public/palindrome.html [hunk 1]
```diff
@@ -5,10 +5,49 @@
...
+    <div id="div2">
+    <b><h2 style="font-family: 'Poppins', sans-serif;">-Let's test a string if it is a palindrome or not- </h2></b>
+    <form id="form" onsubmit="fun()">
+        <input type="text" id="fld"><br>
+        <input type="button" value="check" id="btn" onclick="fun()">
+        <input type="button" value="Reset" id="btn" onclick="myFunction()">
+        <p id="result"></p>
+    </form>
 	<div id="bg"></div>
 	<div id="def" class="def">
```
Comment (991rajat) [inline public/palindrome.html:52]: This is same, I think removing this will fix it. Look this at once.

**Review by 991rajat — COMMENTED** [review]
Fix the problems please

File: public/palindrome.html (hunk 1 above)
Comment (Aarti002) [reply public/palindrome.html:52]: i didn't get it, @991rajat which part should i remove?

**Comment by Aarti002** [comment]
@991rajat can you review now, this time it might work properly.
Thank you!

**Review by 991rajat — APPROVED** [review]
Looks Good.

**Pull request merged**